tecken/apps.py

```diff
@@ -19,3 +20,15 @@ def ready(self):
         # The app is now ready.
 
         markus.configure(settings.MARKUS_BACKENDS)
+
+        connection = get_redis_connection('store')
+        maxmemory_policy = connection.info()['maxmemory_policy']
+        if maxmemory_policy != 'allkeys-lru':
+            logger.warning(
+                "The 'store' Redis cache was not configured to be an "
+                "LRU cache because the maxmemory-policy was set to '{}'. "
+                "Now changed to 'allkeys-lru'.".format(
```

Comment:
You are right. I'm glad I tested this. In ElastiCache Redis you can't send any `CONFIG` commands. Not even to *read* config. 
So I'm going to document how you have to do it on the infra side. 
The code here above will be changed slightly so it only tries to do this if `settings.DEBUG`.